frontend/src/components/CreateCollectionModal/components/Content/components/AddLink/index.tsx

```diff
@@ -37,15 +34,14 @@ const LinkTypes: FC<Props> = ({ handleClick }) => {
   );
 };
 
-const AddLink: FC<Props> = ({ handleClick }) => {
+const AddLink: FC<AddLinkProps> = ({ handleClick, Button }) => {
   return (
     <Popover
       content={<LinkTypes handleClick={handleClick} />}
       position={Position.BOTTOM_LEFT}
+      captureDismiss={true}
```

Comment:
Not setting this led to this ridiculously obfuscated bug where choosing a link type would close the menu that held the edit collection button.  Closing that menu would close the entire edit collection modal

Reply:
interesting TIL! Can you add this as a comment in the code too please?

Reply:
```suggestion
      captureDismiss={true} // Not setting this led to this ridiculously obfuscated bug where choosing a link type would close the menu that held the edit collection button. Closing that menu would close the entire edit collection modal
```